judge/views/organization.py

```diff
@@ -85,18 +85,22 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class OrganizationUsers(OrganizationDetailView):
+class OrganizationUsers(QueryStringSortMixin, OrganizationDetailView):
     template_name = 'organization/users.html'
+    all_sorts = frozenset(('points', 'problem_count', 'rating', 'performance_points', 'user__username'))
```

Comment:
Allowing username sorting doesn't fall in line with user list sorting elsewhere. Not sure if we should deviate from that for organizations.

Reply:
I currently have another previous pr [open](https://github.com/DMOJ/online-judge/pull/1523) to ensure consistency, once that is resolved I'll make this consistent with it and un-mark this as a draft.

Reply:
Username sorting is no longer allowed.